store/test.util.ts

```diff
@@ -0,0 +1,5 @@
+export const app = {
```

Comment:
What do you think about moving this object to the test files that use it?  I know it's duplication but I don't think we gain much with this abstraction.

Reply:
The objective was to determine whether the schema entity definition could be declared in a separate file and then imported. To maintain a clean directory structure, I believe it's best to eliminate this particular test since it doesn't carry enough meaningful information to justify the creation of a new file and so on.